criu/proc_parse.c

```diff
@@ -173,8 +173,7 @@ static void parse_vma_vmflags(char *buf, struct vma_area *vma_area)
 	 * only exception is VVAR area that mapped by the kernel as
 	 * VM_IO | VM_PFNMAP | VM_DONTEXPAND | VM_DONTDUMP
 	 */
-	if (io_pf && !vma_area_is(vma_area, VMA_AREA_VVAR) &&
-	    !vma_entry_is(vma_area->e, VMA_FILE_SHARED))
+	if (io_pf && !vma_area_is(vma_area, VMA_AREA_VVAR))
```

Comment:
> I am not sure about how to handle systemcfg file but the current solution looks like a workaround so revert it.

This doesn't work this way. Any new changes must not affect existing users. If you explain how it breaks your plugin, we will be able to think together about how to fix this issue.

Reply:
I saw this issue a couple of months ago when this change was first introduced. Since then, our plugin and criu has advanced and on the latest code, I didn't see this issue again. I will drop this change for now but will keep an eye on it and will re-open it if I ever see this again.